src/data.py

```diff
@@ -96,16 +101,16 @@ def get_items(self) -> list:
     def set_items(self, items: list):
         """Set all managed items."""
         for item_ in items:
-            self.append(item_)
+            self._add_item(item_)
         self._items = {item_.get_uuid(): item_ for item_ in items}
 
-    def append(self, item_):
+    def _add_item(self, item_):
         """Append items to self."""
         self._connect_to_item(item_)
         self._items[item_.get_uuid()] = item_
         self.notify("items")
 
-    def pop(self, key):
+    def _delete_item(self, key):
```

Comment:
I was actually considering renaming this mehtod either way, as a `pop` action in Python also returns the item that is deleted. This method does not, so it's a bit of misleading name to call this pop.